.goreleaser.yaml

```diff
@@ -24,11 +24,11 @@ archives:
   # GitHub release should contain the raw binaries (no zip or tar.gz)
 - format: binary
   # Name format should match the curl install script
-  name_template: "{{ .ProjectName }}-{{ .Os }}-{{ .Arch }}"
-  replacements:
```

Comment:
`replacements` is deprecated, I migrated the logic to `name_template` (this is the recommended way according to the docs)